api/src/main/java/org/confeti/service/dto/Report.java

```diff
@@ -30,6 +33,7 @@
 
     private static final long serialVersionUID = 1L;
 
+    @JsonIgnore
```

Comment:
@JsonIgnore is used to ignore the logical property used in serialization and deserialization. But we need this field in serialization. And I think we should add the test that this field is ignored during the deserialization.